docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md

```diff
@@ -0,0 +1,32 @@
+# Aliquot Ensemble Somatic Variant Merging and Masking
+
+## Description ##
+
+Aliquot Ensemble Somatic Variant Merging and Masking is a somatic variant calling pipeline used in GDC whole exome sequencing (WXS) and targeted sequencing harmonization.
```

Comment:
Technically somatic variants aren't called, so I would recommend changing it from "a somatic variant calling pipeline" to "an aggregation pipeline"

Reply:
changed